packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -341,21 +350,39 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
 class FlutterInspectorSettingsDialog extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    preferences.inspector.loadCustomPubRootDirectories();
+    final theme = Theme.of(context);
     return DevToolsDialog(
       title: dialogTitleText(Theme.of(context), 'Flutter Inspector Settings'),
-      content: Column(
-        mainAxisSize: MainAxisSize.min,
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          CheckboxSetting(
-            notifier: preferences.inspector.hoverEvalModeEnabled
-                as ValueNotifier<bool?>,
-            title: 'Enable hover inspection',
-            description:
-                'Hovering over any widget displays its properties and values.',
-            gaItem: analytics_constants.inspectorHoverEvalMode,
-          ),
-        ],
+      content: Container(
+        width: defaultDialogWidth,
+        height: scaleByFontFactor(400.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            ...dialogSubHeader(
+              Theme.of(context),
+              'General',
+            ),
+            CheckboxSetting(
+              notifier: preferences.inspector.hoverEvalModeEnabled
+                  as ValueNotifier<bool?>,
+              title: 'Enable hover inspection',
+              description:
+                  'Hovering over any widget displays its properties and values.',
+              gaItem: analytics_constants.inspectorHoverEvalMode,
+            ),
+            const SizedBox(height: denseSpacing),
+            ...dialogSubHeader(Theme.of(context), 'Package Directories'),
+            Text(
+              'widgets in these directories will show up in your summary tree',
```

Comment:
r/widgets/Widgets and add a period at the end.

We should also add something like `(e.g. /absolute/path/to/myPackage)` to demonstrate what the path should look like.